E2E test "test/e2e/compatibility/cp_compatibility_kubernetes_multizone.go" can't download Kuma 0.6.3

### Summary

Not sure whether it's reasonable for this test to expect Kuma 90.6.3 is available forever, but it seems to be gone now. Possible that I'm not doing some test setup correctly.

```
=== RUN   TestE2ECompatibility
RegisterFailHandlerWithT is deprecated.  Please use RegisterFailHandler or RegisterTestingT instead.
Running Suite: E2E Compatibility Suite
======================================
Random Seed: 1632972911
Will run 2 of 2 specs

Test Kubernetes Multizone Compatibility should sync resources between new global and old zone 2021-09-30T13:35:11+10:00 client.go:33: Configuring kubectl using config file /home/jpeach/.kube/kind-kuma-1-config with context 
Test Kubernetes Multizone Compatibility should sync resources between new global and old zone 2021-09-30T13:35:11+10:00 client.go:33: Configuring kubectl using config file /home/jpeach/.kube/kind-kuma-1-config with context 
Test Kubernetes Multizone Compatibility should sync resources between new global and old zone 2021-09-30T13:35:11+10:00 client.go:33: Configuring kubectl using config file /home/jpeach/.kube/kind-kuma-1-config with context 
Test Kubernetes Multizone Compatibility should sync resources between new global and old zone 2021-09-30T13:35:11+10:00 logger.go:66: Running command helm with args [install --kubeconfig /home/jpeach/.kube/kind-kuma-1-config --namespace kuma-system --version 0.6.3 --set controlPlane.defaults.skipMeshCreation=false --set controlPlane.envVars.KUMA_BOOTSTRAP_SERVER_API_VERSION=v3 --set controlPlane.globalZoneSyncService.type=NodePort --set controlPlane.image.repository=kuma-cp --set controlPlane.mode=global --set dataPlane.image.repository=kuma-dp --set dataPlane.initImage.repository=kuma-init --set global.image.registry=kumahq kuma-ujedmx kuma/kuma]
Test Kubernetes Multizone Compatibility should sync resources between new global and old zone 2021-09-30T13:35:11+10:00 logger.go:66: Error: failed to download "kuma/kuma" at version "0.6.3" (hint: running `helm repo update` may help)
• Failure in Spec Setup (BeforeEach) [0.123 seconds]
Test Kubernetes Multizone Compatibility
/home/jpeach/upstream/konghq/kuma/test/e2e/compatibility/cp_compatibility_kubernetes_multizone_test.go:9
  should sync resources between new global and old zone [BeforeEach]
  /home/jpeach/upstream/konghq/kuma/test/e2e/compatibility/cp_compatibility_kubernetes_multizone.go:105

  Unexpected error:
      <*shell.ErrWithCmdOutput | 0xc000a1a858>: {
          Underlying: <*exec.ExitError | 0xc000d223c0>{
              ProcessState: {
                  pid: 2442827,
                  status: 256,
                  rusage: {
                      Utime: {Sec: 0, Usec: 44084},
                      Stime: {Sec: 0, Usec: 15744},
                      Maxrss: 56552,
                      Ixrss: 0,
                      Idrss: 0,
                      Isrss: 0,
                      Minflt: 3445,
                      Majflt: 389,
                      Nswap: 0,
                      Inblock: 66800,
                      Oublock: 0,
                      Msgsnd: 0,
                      Msgrcv: 0,
                      Nsignals: 0,
                      Nvcsw: 639,
                      Nivcsw: 26,
                  },
              },
              Stderr: nil,
          },
          Output: {
              stdout: {
                  Lines: nil,
                  merged: {
                      Mutex: {state: 0, sema: 0},
                      Lines: [
                          "Error: failed to download \"kuma/kuma\" at version \"0.6.3\" (hint: running `helm repo update` may help)",
                      ],
                  },
              },
              stderr: {
                  Lines: [
                      "Error: failed to download \"kuma/kuma\" at version \"0.6.3\" (hint: running `helm repo update` may help)",
                  ],
                  merged: {
                      Mutex: {state: 0, sema: 0},
                      Lines: [
                          "Error: failed to download \"kuma/kuma\" at version \"0.6.3\" (hint: running `helm repo update` may help)",
                      ],
                  },
              },
              merged: {
                  Mutex: {state: 0, sema: 0},
                  Lines: [
                      "Error: failed to download \"kuma/kuma\" at version \"0.6.3\" (hint: running `helm repo update` may help)",
                  ],
              },
          },
      }
      error while running command: exit status 1; Error: failed to download "kuma/kuma" at version "0.6.3" (hint: running `helm repo update` may help)
  occurred

  /home/jpeach/upstream/konghq/kuma/test/e2e/compatibility/cp_compatibility_kubernetes_multizone.go:55
------------------------------
```

--- a/test/e2e/compatibility/cp_compatibility_kubernetes_multizone.go
+++ b/test/e2e/compatibility/cp_compatibility_kubernetes_multizone.go
@@ -5,6 +5,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/gruntwork-io/terratest/modules/helm"
 	"github.com/gruntwork-io/terratest/modules/k8s"
 	"github.com/gruntwork-io/terratest/modules/random"
 	. "github.com/onsi/ginkgo"
@@ -25,6 +26,24 @@ func CpCompatibilityMultizoneKubernetes() {
 	var zoneCluster Cluster
 	var zoneDeployOptsFuncs = KumaZoneK8sDeployOpts
 	var zoneReleaseName string
+
+	// Ensure that the upstream Kuma help repository is configured
+	// and refreshed. This is needed for helm to be able to pull the
+	// OldChart version of the Kuma helm chart.
+	BeforeSuite(func() {
+		t := NewTestingT()
+		opts := helm.Options{}
+
+		// Adding the same repo multiple times is idempotent. The
+		// `--force-update` flag prevents heml emitting an error
+		// in this case.
+		_, err := helm.RunHelmCommandAndGetOutputE(t, &opts,
+			"repo", "add", "--force-update", "kuma", "https://kumahq.github.io/charts")
+		Expect(err).To(Succeed())
+
+		_, err = helm.RunHelmCommandAndGetOutputE(t, &opts, "repo", "update")
+		Expect(err).To(Succeed())
+	})
 
 	BeforeEach(func() {
 		// Global CP

